srcpkgs/cqrlog/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'cqrlog'
+pkgname=cqrlog
+version=2.3.0
+revision=1
+archs="x86_64 i686"
```

Comment:
Please leave a comment on reason.

Reply:
Done.

Reply:
That's not considered bad style by some maintainers, since the package itself is not arch-restricted when one of its dependencies is which might lead to dangling restrictions once a dependency is patched to support more archs. Just let it fail due to lazarus (currently) not working on other than mentioned archs?

Reply:
Packages were marked this way because otherwise build started and failed on builders again  and again. Can someone managing builders confirm that we should still actively prevent to build packages with missing deps?